com.github.AberothCommunity.Aberoth.yaml

```diff
@@ -0,0 +1,44 @@
+app-id: com.github.AberothCommunity.Aberoth
```

Comment:
the app-id must be starting with `io.github.`

see https://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
Actually since `https://github.com/AberothCommunity` doesn't exist, it's not a valid id.

Reply:
sorry, I didn't notice that. I can't use aberoth-community because it throws an error, what should I do in this situation?

Reply:
com.aberoth.AberothClient, com.aberoth.Aberoth, com.aberoth.AberothCommunity etc.